libs/dh/metering-point/shell/src/lib/dh-metering-point-shell.module.ts

```diff
@@ -37,12 +37,14 @@ const routes: Routes = [
   { path: 'search', component: DhMeteringPointSearchComponent },
   {
     path: `:${dhMeteringPointIdParam}`,
-    component: DhMeteringPointOverviewComponent,
     canActivate: [DhMeteringPointOverviewGuard],
-  },
-  {
-    path: `:${dhMeteringPointIdParam}/child/:child-id`,
-    component: DhMeteringPointChildOverviewComponent,
+    children: [
+      { path: '', component: DhMeteringPointOverviewComponent },
```

Comment:
issue: this route must have `pathMatch: 'full'`, or the following child route can never be matched